README.md

```diff
@@ -77,9 +77,11 @@ rustup component add rust-src llvm-tools-preview
 sudo apt install qemu-system-aarch64
 ```
 
+Additionally you will need a version of the [aarch64-none-elf](https://developer.arm.com/Tools%20and%20Software/GNU%20Toolchain) toolchain installed. To do this, download the correct release of `aarch64-none-elf` onto your computer, unpack it, then export the `bin` folder to path.
+
```

Comment:
Might be worth adding that if you're on NixOS you can get this by running:

```bash
nix shell nixpkgs#pkgsCross.aarch64-embedded.stdenv.cc nixpkgs#pkgsCross.aarch64-embedded.stdenv.cc.bintools
```

That's how I do it.  I should probably create a `flake.nix` to set up any tools that people will need.

Reply:
Is it actually possible to not use objcopy from the aarch64 toolchain whatsoever?